Tensile/SolutionStructs.py

```diff
@@ -2499,11 +2499,24 @@ def assignDerivedParameters(state):
       else:
         tva = totalElementsA // state["GlobalReadVectorWidth"]
         tvb = totalElementsB // state["GlobalReadVectorWidth"]
-        if not Solution.setGlobalLoadVectorWidth(state, "A", tva):
+        if not Solution.setGlobalLoadVectorWidth(state, "A", tva, state["GlobalReadVectorWidth"]):
           validDepthU = False
-        if not Solution.setGlobalLoadVectorWidth(state, "B", tvb):
+        if not Solution.setGlobalLoadVectorWidth(state, "B", tvb, state["GlobalReadVectorWidth"]):
           validDepthU = False
 
+        if state["EnableMatrixInstruction"] and state["GlobalLoadVectorWidthA"]:
+          partialA = state["ProblemType"]["TLUA"] and (state["AssertFree0ElementMultiple"]%state["GlobalLoadVectorWidthA"] != 0)
+          if partialA and state["GlobalLoadVectorWidthA"] > state["MIOutputVectorWidth"]:
+            #reduce GLVA if GLVA larger than MIOVW
```

Comment:
In partial case, tensile should choose mioutputvectorWidth over GLVW; is this for to avoid partial altogether or write shift-ptr logic to work?

Reply:
Yes.
For A matrix, shift-ptr only work when GLVW  <= MIoutputVectorWidth.
For B matrix, shift-ptr can work when GLVW <= MatrixInstN.

This code to reduce GLVWA size to MIOutputVectorWidth. Later shift-ptr checking will not reject this solution.
It let GLVWB has chance to use higher GLVW. Such as GLVWB=8.

Reply:
> In partial case, tensile should choose mioutputvectorWidth over GLVW; is this for to avoid partial altogether or write shift-ptr logic to work?
If without this patch, tensile only reject the solution that GLVW > MIOVW. It wouldn't choose MIOVW over GLVW.